### ISSUE TITLE
(Feature request) DirectML support

### ISSUE THREAD

**Issue by NotNite** [bitsandbytes-foundation/bitsandbytes#194]
I have an AMD GPU, so my way of getting around not having CUDA is by using DirectML. Unfortunately, this depends on CUDA APIs (and is Linux only), so I can't use it.

**Comment by RoARene317** [comment]
Just use Linux, make sure your GPU support ROCm. Case closed.

**Comment by yoda5477** [comment]
This would be greatly appeciated now that it is needed for flux nf4. not sure how complexe to solve the current issue link to only cuda compatible

**Comment by ethrx** [comment]
Can we re-open this?

It's a valid use-case.

**Comment by matthewdouglas** [comment]
@ethrx I agree, especially for consumer AMD GPUs not otherwise supported in ROCm. I'll reopen this, but we won't be able to get to this right away.
